src/gitutil.js

```diff
@@ -137,6 +137,10 @@ exports.pendingChangesExist = function * () {
 exports.gitCheckout = function * (branchName) {
     var curBranch = yield gitutil.retrieveCurrentBranchName(true);
     if (curBranch !== branchName) {
+        // EXTRA WORKAROUND SOLUTION for package.json,
```

Comment:
Is there an issue somewhere that describes what problem is happening here?

Reply:
Added TODO comments to raise issues